src/command-line-handling.cc

```diff
@@ -407,6 +411,114 @@ void gq_delay(GtkApplication *, GApplicationCommandLine *app_command_line, GVari
 	options->slideshow.delay = static_cast<gint>(n * 10.0 + 0.01);
 }
 
+void gq_duplicates_process(GtkApplication *, GApplicationCommandLine *, GVariantDict *, GList *file_list)
+{
+	std::map<std::string, std::unique_ptr<pic_equiv>> pics;
```

Comment:
`unique_ptr` seems redundant.

Reply:
Thanks for the review @qarkai 
I addressed the other comments, but with my rudimentary C++ skills I cannot see how to address this one?
- replacing with `pic_equiv *` would leak memory, right?
- replacing with `pic_equiv` does not work because the assignment operator is deleted:
  ```
../src/command-line-handling.cc: In function ‘void {anonymous}::gq_duplicates_process(GtkApplication*, GApplicationCommandLine*, GVariantDict*, GList*)’:
../src/command-line-handling.cc:422:42: error: use of deleted function ‘pic_equiv& pic_equiv::operator=(const pic_equiv&)’
  422 |                 pics[name] = pic_equiv(fd);
```
  and it's deleted because of the `sim` member.

  It also seems to me like using a pointer is better than copying the sim structure around...

Reply:
I see. `ImageSimilarityData` lacks constructors and moving operator. Ok. let's stay with `unique_ptr` for now.